packages/sdk/README.md

```diff
@@ -41,13 +41,27 @@ TeamsFx SDK is built to be used in browser and NodeJS environment. Common scenar
 - Azure Function
 - Teams bot
 
-### Create and authenticate a service like `MicrosoftGraphClient`
+### Create and authenticate a service like `createMicrosoftGraphClientWithCredential` or `createMicrosoftGraphClient`
```

Comment:
`MicrosoftGraphClient` is created instance class. `createMicrosoftGraphClientWithCredential` is the API name.
I think it can either be unchanged or "Create and authenticate a service using `createMicrosoftGraphClientWithCredential` or `createMicrosoftGraphClient`"

Reply:
Sure, thanks~